### PR TITLE
New arrival voice

### PR DESCRIPTION
**Pull request by MajManatee**
# Document the changes in your pull request
No longer will it says "TG Station" and leave people confused every time you late join.

A 99% chance to play the normal, welcome sound
and a fun, 1% chance to have a... special flair on the end.
as per special request.

https://user-images.githubusercontent.com/74586965/208832878-8006e4b8-32ee-45ec-b377-514bde173156.mp4

https://user-images.githubusercontent.com/74586965/208832879-a5442495-c7cf-4d67-993a-34662ef2f7fa.mp4

# Changelog

:cl:  @MajManatee and @cuackles 
rscadd: Chance for a alternate arrival sound
soundadd: New arrival sound(s)
sounddel: old sound out the window
/:cl:

### CODE REVIEW

**Comment by Cartlord** [comment]
sounds like "rogstation"

**Comment by ShoziOffical** [comment]
Finally

**Comment by Moltijoe** [comment]
Maybe it's just me, but i hear a weird sound artifact at the first I part of "arriving"

**Comment by Cartlord** [comment]
Yeah, there's definitely a click there.

**Comment by cuackles** [comment]
> Maybe it's just me, but i hear a weird sound artifact at the first I part of "arriving"

Yep it's there i didnt notice it at first so i will have to fix it tomorrow

**Comment by ReddicusDragon** [comment]
pain

**Comment by ChuChu42** [comment]
it sound good 
remove the fart part

File: code/modules/mob/dead/new_player/new_player.dm [hunk 1]
```diff
@@ -399,8 +399,11 @@
 		if(!arrivals_docked)
 			var/atom/movable/screen/splash/Spl = new(character.client, TRUE)
 			Spl.Fade(TRUE)
-			character.playsound_local(get_turf(character), 'sound/voice/ApproachingTG.ogg', 25)
-
+			switch(rand(1, 100))
+				if(1 to 99)
+					character.playsound_local(get_turf(character), 'sound/voice/ApproachingYOG.ogg', 25)
+				if(100)
+					character.playsound_local(get_turf(character), 'sound/voice/ApproachingYOGbutepic.ogg', 25)
```
Comment (Moltijoe) [inline code/modules/mob/dead/new_player/new_player.dm:406]: sorta weird way of doing this, but i guess it works

**Review by Moltijoe — COMMENTED** [review]
Code checks out, sound needs work

File: code/modules/mob/dead/new_player/new_player.dm (hunk 1 above)
Comment (MajManatee) [reply code/modules/mob/dead/new_player/new_player.dm:406]: sound updated as of next push

**Comment by JamieD1** [comment]
I reckon we just use the original noise and then get yogurt to say Yog and then clip it over TG, would be more funny.

**Comment by MajManatee** [comment]
its been at least 5 years, and nobody has made that "simple change" yet. At least this is SOMETHING

**Comment by MajManatee** [comment]
uhh no longer relevant soon. :)

**Pull request closed without merging**